oap-server/server-starter/src/main/resources/ui-initialized-templates/redis/redis-service.json

```diff
@@ -311,6 +311,37 @@
                   }
                 }
               ]
+            },
+            {
+              "name":"Slow Statements",
+              "children":[
+                {
+                  "x":0,
+                  "y":0,
+                  "w":24,
+                  "h":31,
+                  "i":"0",
+                  "type":"Widget",
+                  "metricTypes":[
+                    "readRecords"
+                  ],
+                  "metrics":[
+                    "top_n_database_statement"
+                  ],
+                  "graph":{
+                    "type":"TopList",
+                    "color":"purple"
+                  },
+                  "widget":{
+                    "title":"Slow Statements (ms)"
```

Comment:
Are these statements or commands? I think Redis prefer `command`?

Reply:
Yes,these should be commands.